builder.md

```diff
@@ -81,14 +90,16 @@ Each template repository MUST have a templates folder containing a config.yaml a
 These files have the following structure:
 
     default:
-        extension: .file-extension
-        output: output-directory-name
+      extension: .file-extension
+      output: output-directory-name
 
     additional:
-        extension: .file-extension
-        output: output-directory-name
+      styles: [base16]
+      filename: "output-directory-name/{{ scheme-style }}-{{ scheme-slug }}.file-extension"
```

Comment:
Seems weird that style here is a variable while extension is not, but maybe it'll make sense after you explain the array of styles. :)

Reply:
I'd honestly prefer to drop `extension` and `output` entirely, but that wouldn't be backwards compatible with templates (which is something we've made a goal for a very long time).

Reply:
Deprecate them now and make them all variables...?  Removing them can come later or never, but we can suggest people in the larger base17+ ecosystem start doing it the new way.

Reply:
I think I'm going to focus on the "filename" and specify that extension & output are legacy from base16 and should not be used long-term. I just want to avoid breaking templates using one of our builders if at all possible.

Reply:
Yeah, I think we just said the same thing, lol.